cobbler/utils.py

```diff
@@ -221,6 +221,32 @@ def is_systemd() -> bool:
     return False
 
 
+def is_supervisord() -> bool:
+    """
+    Return whether or not this system uses systemd.
+
+    This method currently checks if the path ``/usr/lib/systemd/systemd`` exists.
+    """
+    if os.path.exists("/usr/bin/supervisord"):
+        return True
+    return False
+
+
+def service_restart(service_name):
+
+    if is_systemd():
+        restart_command = "systemctl restart " + service_name
+    # elif is_supervisord():
+    # FIXME what needs to be done for supervisord?
```

Comment:
`supervisorctl restart <UNIT>`

Reply:
and for status in check.py? Is is supervisorctl status <service>?

Reply:
Yes: http://supervisord.org/running.html#supervisorctl-actions